apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -307,6 +307,7 @@ events:
     properties:
       - name: code
         summary: If `success` is `false`, the error code is available.
+        type: Number
```

Comment:
I'm honestly not even sure this event can be fired with an error...

Reply:
AFAIK on Android `code` is always `0`
(https://github.com/appcelerator/titanium_mobile/blob/b295e6321ba38e29612b51d93c1518e87e6fffee/android/modules/geolocation/src/java/ti/modules/titanium/geolocation/TiCompass.java#L135-L136)